etcdserver/api/etcdhttp/metrics.go

```diff
@@ -57,34 +58,56 @@ func newHealthHandler(srv *etcdserver.EtcdServer) http.HandlerFunc {
 			return
 		}
 		h := checkHealth(srv)
-		d := []byte(fmt.Sprintf(`{"health": "%v"}`, h.Health))
+		d := h.marshal()
 		if !h.Health {
-			http.Error(w, string(d), http.StatusServiceUnavailable)
+			http.Error(w, d, http.StatusServiceUnavailable)
 			return
 		}
 		w.WriteHeader(http.StatusOK)
-		w.Write(d)
+		w.Write([]byte(d))
 	}
 }
 
 type health struct {
-	Health bool `json:"health"`
+	Health bool     `json:"health"`
+	Errors []string `json:"errors"`
+}
+
+func (h health) marshal() string {
+	s := fmt.Sprintf(`{"health": "%v"`, h.Health)
```

Comment:
i am not sure if this is that important... the previous "true" is a mistake. we should change it to either "health": "ok" or "health": true. @heyitsanthony opinion?